src/mame/konami/nwk-tr.cpp

```diff
@@ -654,12 +607,41 @@ static INPUT_PORTS_START( nwktr )
 	PORT_BIT( 0xfff, 0x000, IPT_PEDAL2 ) PORT_NAME("Brake Pedal") PORT_MINMAX(0x000, 0xfff) PORT_SENSITIVITY(100) PORT_KEYDELTA(60)
 
 	PORT_START("ANALOG4")
-	PORT_BIT( 0xfff, 0x000, IPT_AD_STICK_Y ) PORT_NAME("Handbrake Lever") PORT_MINMAX(0x000, 0xfff) PORT_SENSITIVITY(100) PORT_KEYDELTA(60) PORT_REVERSE
+	PORT_BIT( 0xfff, 0x000, IPT_CUSTOM )
 
 	PORT_START("ANALOG5")
-	PORT_BIT( 0xfff, 0x000, IPT_PEDAL3 ) PORT_NAME("Clutch Pedal") PORT_MINMAX(0x000, 0xfff) PORT_SENSITIVITY(100) PORT_KEYDELTA(60)
+	PORT_BIT( 0xfff, 0x000, IPT_CUSTOM )
```

Comment:
Please replace the `IPT_CUSTOM` placeholders with `IPT_UNKNOWN`.  You should only use `IPT_CUSTOM` with custom handler callbacks.